package.json

```
@@ -1,16 +1,16 @@
 {
   "name": "mongoose-delete",
-  "version": "1.0.2",
+  "version": "1.1.0-canary-1",
```

🤔 I don't think we need to canary this one
```suggestion
  "version": "1.1.0",
```

---

You've got to change this one to merge, but you can certainly send it up to npm with the canary as is